### PR TITLE
Accessibiity: Labels for nav bar icons

### PR DESCRIPTION
**Pull request by akashkj**
## Product Description
Add labels for accessibility to global navigation bar icons in HQ.
image

## Technical Summary
Added `aria-label` attribute to the relevant HTML tags.
https://dimagi-dev.atlassian.net/browse/USH-2002

## Feature Flag
NA

### Safety story
Local environment testing with Linux's screen reader

https://user-images.githubusercontent.com/14318173/214167830-f6970214-5a28-4b09-ae81-e014f2c27b3c.mp4

### Automated test coverage
NA

### QA Plan
NA

### Migrations
NA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by Robert-Costello — APPROVED** [review]

**Comment by Robert-Costello** [comment]
Looks good! We often send these accessibility changes through QA to check for unexpected behavior, but this one seems pretty straightforward.

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
 <ul class="nav navbar-nav" role="menu">
     <li class="dropdown">
         <a href="#" class="dropdown-toggle dropdown-toggle-with-icon track-usage-link"
-           data-toggle="dropdown" data-category="Nav Bar" data-action="Click Gear Icon">
+           data-toggle="dropdown" data-category="Nav Bar" data-action="Click Gear Icon"
+           aria-label="Settings">
             <i class="fa fa-cog nav-main-icon"></i> <span class="responsive-label">{% trans "Manage" %}</span>
```
Comment (stephherbers) [inline corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html:11]: Is there a reason for the mismatch in "Manage" and "Settings"? (I honestly maybe like settings more, but could cause for some confusion)

**Comment by orangejenny** [comment]
These should be translated with `trans_html_attr`, right?

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html (hunk 1 above)
Comment (stephherbers) [reply corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html:11]: Oh cool I didn't see that-- "Manage Settings" is a great idea

Comment (akashkj) [reply corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html:11]: Are we finalizing on "Manage Settings" then?

**Comment by akashkj** [comment]
Updated with translation tag

**Review by Robert-Costello — APPROVED** [review]

**Pull request merged**